src/containers/Layout/styles.sass

```diff
@@ -0,0 +1,10 @@
+.map-container
```

Comment:
I find SCSS syntax redundant in comparison with SASS. However, react-toolbox components are writen in SCSS. We may use it if you like.